wgsl/index.bs

```diff
@@ -9035,8 +9035,8 @@ shader that goes beyond the specified limits.
 # Memory # {#memory}
 
 In WGSL, a value of [=storable=] type may be stored in memory, for later retrieval.
-This section describes the structure of memory, and how WGSL types are used to
-describe the contents of memory.
+This section describes the structure of memory, and the semantics of operations accessing memory.
```

Comment:
This was the nit I mentioned in passing in #3851.
This section no longer defines the *types* for memory. So refer back to the "memory views" section.